cvat-ui/src/components/annotation-page/attribute-annotation-workspace/attribute-annotation-sidebar/attribute-annotation-sidebar.tsx

```diff
@@ -0,0 +1,300 @@
+        ));
+    const activeAttribute = activeObjectState
+        ? labelAttrMap[activeObjectState.label.id]
+        : null;
+
+    if (activeObjectState) {
+        const attribute = labelAttrMap[activeObjectState.label.id];
+        if (attribute && attribute.id !== activatedAttributeID) {
+            activateObject(activatedStateID, attribute ? attribute.id : null);
+        }
+    } else if (states.length) {
+        const attribute = labelAttrMap[states[0].label.id];
+        activateObject(states[0].clientID, attribute ? attribute.id : null);
+    }
+
+    const nextObject = (step: number): void => {
+        if (states.length) {
+            const index = states.indexOf(activeObjectState);
+            let nextIndex = index + step;
+            if (nextIndex > states.length - 1) {
+                nextIndex = 0;
+            } else if (nextIndex < 0) {
+                nextIndex = states.length - 1;
+            }
+            if (nextIndex !== index) {
+                const attribute = labelAttrMap[states[nextIndex].label.id];
+                activateObject(states[nextIndex].clientID, attribute ? attribute.id : null);
+            }
+        }
+    };
+
+    const nextAttribute = (step: number): void => {
+        if (activeObjectState) {
+            const { label } = activeObjectState;
+            const { attributes } = label;
+            if (attributes.length) {
+                const index = attributes.indexOf(activeAttribute);
+                let nextIndex = index + step;
+                if (nextIndex > attributes.length - 1) {
+                    nextIndex = 0;
+                } else if (nextIndex < 0) {
+                    nextIndex = attributes.length - 1;
+                }
+                if (index !== nextIndex) {
+                    const updatedLabelAttrMap = { ...labelAttrMap };
+                    updatedLabelAttrMap[label.id] = attributes[nextIndex];
+                    setLabelAttrMap(updatedLabelAttrMap);
+                }
+            }
+        }
+    };
+
+    useEffect(() => {
+        if (document.activeElement instanceof HTMLElement) {
+            document.activeElement.blur();
+        }
+    }, []);
+
+    const siderProps: SiderProps = {
+        className: 'attribute-annotation-sidebar',
+        theme: 'light',
+        width: 300,
+        collapsedWidth: 0,
+        reverseArrow: true,
+        collapsible: true,
+        trigger: null,
+    };
+
+    const keyMap = {
+        NEXT_ATTRIBUTE: {
+            name: 'Next object',
+            description: 'Go to the next object. Use Ctrl when inside an input field',
+            sequences: ['ArrowDown', 'Ctrl+ArrowDown'],
+            action: 'keydown',
+        },
+        PREVIOUS_ATTRIBUTE: {
+            name: 'Previous object',
+            description: 'Go to the previous object. Use Ctrl when inside an input field',
+            sequences: ['ArrowUp', 'Ctrl+ArrowUp'],
+            action: 'keydown',
+        },
```

Comment:
Suppose you meant next/previous attribute here. Also i noticed that +Ctrl combinations doesn't work (for example `Shift+Ctrl+Tab`/`Ctrl+Tab` just switch you between tabs, but regular combinations works fine inside focused input fields.

Reply:
Sure, old code.